fastlabel/__init__.py

```diff
@@ -3772,6 +3772,45 @@ def copy_project(self, project_id: str) -> None:
         endpoint = "projects/copy"
         return self.api.post_request(endpoint, payload=payload)
 
+    # Project Tags
+
+    def get_tags(
```

Comment:
```suggestion
    def get_project_tags(
```
タスクにひもづくタグと区別するために、projectつけた方が良さそう